public/docs/_examples/dependency-injection/dart/lib/app_component_1.dart

```diff
@@ -1,17 +1,17 @@
 // Early versions
 
 // #docregion
-import 'package:angular2/core.dart';
+import 'package:angular2/angular2.dart';
```

Comment:
Why are the imports changed to `angular2/angular2.dart`? AFAIR it was suggested not too long ago to prefer `core.dart`.